thinkhazard/static/js/report.js

```diff
@@ -39,7 +39,9 @@
 
   // change mouse cursor when over division
   map.on('pointermove', function(e) {
-    var feature = map.forEachFeatureAtPixel(e.pixel, filterFn);
+    var feature = map.forEachFeatureAtPixel(e.pixel, filterFn, {
+      hitTolerance: 3
```

Comment:
Would be better to set this value in a single place, rather than repeating 3 times.